ui/src/app/app.module.ts

```diff
@@ -15,7 +15,37 @@ import { ServicesModule } from './service/services.module';
 import { SharedModule } from './shared/shared.module';
 import { NavbarModule } from './views/navbar/navbar.module';
 
-let ngModule: NgModule = {
+export let errorFactory = () => {
+    if ((<any>window).cds_sentry_url) {
+        class RavenErrorHandler implements ErrorHandler {
+            handleError(err: any): void {
+                console.error(err);
+                Raven.captureException(err);
+            }
+        }
+
+        let tags = {};
+        let userStr = localStorage.getItem('CDS-USER');
```

Comment:
it could be nice to be sure that the username is available in localStorage

Reply:
Done